### DISCUSSION TITLE
data merge

### DISCUSSION THREAD

**Discussion by Cst1610123** [mtex-toolbox/mtex#1416] [Feature Requests]
Could you develop a data merge function？

**Comment by ralfHielscher** [comment]
Could you please elaborate more precisely what this function should do?

1. which data to merge
2. why to merge
3. ...

In general, if you have two lists of data, i.e., orientations `oriList1` ad `oriList2` you can do
```
oriList = [oriList1; oriList2]
```
to merge them.
Ralf.

**Comment by Cst1610123** [comment]
Thank you for your reply. The data I want to merge is *.cpr/crc. I already know how to do it：[ebsd1,ebsd2]. Thank you.
